### PR TITLE
Add getProfile

### PR DESCRIPTION
**Pull request by HugoMaster**

### CODE REVIEW

**Comment by nicolai86** [comment]
I didn't think about this before: what do you think about calling it `getVCard` and not `getProfile`?

**Comment by HugoMaster** [comment]
Right : ) better name

**Comment by nicolai86** [comment]
Can you adjust editProfile as well? And, last change: also remove the avatar parameter,and set the profile attribute in the params object on the JS side?

**Comment by HugoMaster** [comment]
@nicolai86 how about now?

File: RNXMPP/RNXMPPService.m [hunk 1]
```diff
@@ -563,22 +563,33 @@ -(void)fetchRoster {
...
 
     for(NSString *key in params.allKeys){
-        [newvCardTemp setValue:[params objectForKey:key] forKey:key];
-    }
-
-    if (avatar){
-        NSData *avatarData = [[NSData alloc]initWithBase64EncodedString:avatar options:NSDataBase64DecodingIgnoreUnknownCharacters];
-        [newvCardTemp setPhoto:avatarData];
+        NSString *value = [params objectForKey:key];
+        if([[key lowercaseString] containsString:@"photo"]){
```
Comment (nicolai86) [inline RNXMPP/RNXMPPService.m]: why not `isEqualToString` ?

**Comment by nicolai86** [comment]
@HugoMaster apart from one thing, LGTM

File: RNXMPP/RNXMPPService.m (hunk 1 above)
Comment (HugoMaster) [inline RNXMPP/RNXMPPService.m]: For some reason for android and ios it was not working... and always sees it as false.

Comment (nicolai86) [inline RNXMPP/RNXMPPService.m]: the code is not executed for android. well, let's move on for now

**Comment by nicolai86** [comment]
LGTM.

**Pull request merged**